index.html

```diff
@@ -107,7 +117,12 @@ <h1 class="heading-primary">Hey, My name is Mastoora Turkmen</h1>
       </div>
       <div class="home-hero__socials">
         <div class="home-hero__social">
-          <a target="_blank" href="https://www.linkedin.com/in/mastoora-turkmen/" class="home-hero__social-icon-link">
```

Comment:
i adde rel="noopener noreferrer" coz it showing warning ⚠ "noopener" helps prevent tabnabbing attacks, and "noreferrer" enhances privacy and prevents the linked website from knowing the exact URL of the page that referred the user.